src/lib.rs

```diff
@@ -64,6 +64,42 @@ mod contract {
         }
     }
 
+    /// All available methods available for the EVM contract.
+    ///
+    /// This makes it easier to get the correct method strings to use when using
+    /// the library.
+    pub enum Methods {
```

Comment:
I think this should be in tests folder though. It's not going to be used in the contract itself, right?

Reply:
Depends if people are going to use the library as well, if not, then yeah can just live in the tests but, then thats just inflating it a bit. Would users be using the contract module directly at all?

Reply:
No, I don't think the contract would be used as a library. And we do want to avoid inflating the contract size.

Reply:
Makes sense.